wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -173,6 +173,7 @@ public Rotation2d rotateBy(Rotation2d other) {
    * Returns the radian value of the Rotation2d.
    *
    * @return The radian value of the Rotation2d.
+   * @see edu.wpi.first.math.MathUtil#angleModulus(double)
```

Comment:
This doesn't explain why you'd want to use angleModulus() on the return value of this function.